### PR TITLE
Homework/homework 7 strings

### PR DESCRIPTION
**Pull request by KamiloV21**

### CODE REVIEW

File: ConsoleApp1/homework-7-strings/Program.cs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        {
+            StringBuilder alpha = new StringBuilder();
+            StringBuilder num = new StringBuilder();
+            StringBuilder special = new StringBuilder();
+
+            for (int i = 0; i < str.Length; i++)
+            {
+                if (Char.IsDigit(str[i]))
+                    num.Append(str[i]);
+                else if ((str[i] >= 'A' && str[i] <= 'Z') || (str[i] >= 'a' &&str[i] <= 'z'))
```
Comment (sergey-olefir) [inline ConsoleApp1/homework-7-strings/Program.cs:50]: Use char.IsLetter

File: ConsoleApp1/homework-7-strings/Program.cs [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                else
+                    special.Append(str[i]);
+            }
+
+            return special.ToString();
+        }
+
+        static string Sort(string str)
+        {
+            StringBuilder sumbl = new StringBuilder();
```
Comment (sergey-olefir) [inline ConsoleApp1/homework-7-strings/Program.cs:98]: Using StringBuilder is not necessary here

File: ConsoleApp1/homework-7-strings/Program.cs [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            {
+                if (Char.IsDigit(str[i]))
+                    num.Append(str[i]);
+                else if ((str[i] >= 'A' && str[i] <= 'Z') || (str[i] >= 'a' && str[i] <= 'z'))
+                    alpha.Append(str[i]);
+                else
+                    special.Append(str[i]);
+            }
+
+            return num.ToString();
```
Comment (sergey-olefir) [inline ConsoleApp1/homework-7-strings/Program.cs:75]: U have 3 method that doing all the same. It just return different values.

**Review by sergey-olefir — APPROVED** [review]

File: ConsoleApp1/homework-7-strings/Program.cs (hunk 3 above)
Comment (KamiloV21) [reply ConsoleApp1/homework-7-strings/Program.cs:75]: How can i display 3 values ​​at once?

**Pull request merged**